src/components/Filters/Categories/Gender.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
`Gender.js` & `Status.js` are both copies
Remove them and create a generic `<Select>` component